components/sync15/src/client.rs

```diff
@@ -25,16 +29,39 @@ pub struct Sync15StorageClientInit {
     pub tokenserver_url: Url,
 }
 
+fn get_response_timestamp(resp: &mut Response) -> error::Result<ServerTimestamp> {
```

Comment:
Does this need &mut? (either way it won't after the network patch so it doesn't matter much).

Reply:
ah yeah, thanks. `.json` needs mut, but this doesn't.